frontend/templates/cucumber/tests/browser/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/playwright:v1.34.3-jammy as playwright
+
+WORKDIR /tests
+
+COPY package.json package-lock.json ./
+
+RUN npm install
+
+COPY . ./
```

Comment:
```suggestion
COPY . ./
RUN npm install
```
Why not this?

Reply:
It's good practice to do with layers and rebuilding.

If we change `README.md` we don't want to re-run `npm install`. We're far more likely to be adding features than changing `package.json` so this separation allows us to reuse previously built layers.

This matters more for local development or with more consistent use of Docker registries - tbh, I'm not entirely sure how our GHAs and ECR work with layer caching

Reply:
That makes sense - it probably doesn't matter for CI but it's a good practice nonetheless as you said